src/tests/unit/tests/injected/__snapshots__/layered-details-dialog-component.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`LayeredDetailsDialogComponent render component when shadow dom is disabled 1`] = `
+exports[` render component when shadow dom is disabled 1`] = `
```

Comment:
Test name is missing from snapshot - does
```
describe(LayeredDetailsDialogComponent, () => {
```
in ...test.tsx below need to have quotes around `LayeredDetailsDialogComponent`?

Reply:
Good catch. I notice the change but didn't realize the reason: it seems that for classes (actual `class` ...) jest gets the name from the class but in this PR, I changed the component to be a const = NamedSFC and thus jest can't get the name from the const!